angular/lib/src/components/toast/toast.component.ts

```diff
@@ -0,0 +1,221 @@
+				aria-atomic="true"
+			>
+				<ng-template [auSlot]="state().slotStructure" [auSlotProps]="{state: state(), widget}"></ng-template>
+			</div>
+		}`,
+})
+export class ToastComponent extends BaseWidgetDirective<ToastWidget> implements AfterContentChecked {
+	/**
+	 * If `true`, alert can be dismissed by the user.
+	 * The close button (×) will be displayed and you can be notified of the event with the (close) output.
+	 */
+	@Input({alias: 'auDismissible', transform: auBooleanAttribute})
+	dismissible: boolean | undefined;
+
+	/**
+	 * The transition function will be executed when the alert is displayed or hidden.
+	 *
+	 * Depending on the value of CommonAlertProps.animationOnInit, the animation can be optionally skipped during the showing process.
+	 */
+	@Input('auTransition')
+	transition: TransitionFn | undefined;
+
+	/**
+	 * If `true` the alert is visible to the user
+	 */
+	@Input({alias: 'auVisible', transform: auBooleanAttribute})
+	visible: boolean | undefined;
+
+	/**
+	 * If `true`, alert opening will be animated.
+	 *
+	 * Animation is triggered  when the `.open()` function is called
+	 * or the visible prop is changed
+	 */
+	@Input({alias: 'auAnimationOnInit', transform: auBooleanAttribute})
+	animationOnInit: boolean | undefined;
+
+	/**
+	 * If `true`, alert closing will be animated.
+	 *
+	 * Animation is triggered  when clicked on the close button (×),
+	 * via the `.close()` function or the visible prop is changed
+	 */
+	@Input({alias: 'auAnimation', transform: auBooleanAttribute})
+	animation: boolean | undefined;
+
+	/**
+	 * If `true` automatically hides the toast after the delay.
+	 */
+	@Input({alias: 'auAutohide', transform: auBooleanAttribute})
+	autohide: boolean | undefined;
+
+	/**
+	 * Delay in milliseconds before hiding the toast.
+	 */
+	@Input({alias: 'auDelay', transform: auNumberAttribute})
+	delay: number | undefined;
+
+	/**
+	 * Accessibility close button label
+	 */
+	@Input('auAriaCloseButtonLabel') ariaCloseButtonLabel: string | undefined;
+
+	@Input('auSlotDefault') slotDefault: SlotContent<ToastContext>;
+	@ContentChild(ToastBodyDirective, {static: false})
+	slotDefaultFromContent: ToastBodyDirective | null;
+
+	@Input('auSlotStructure') slotStructure: SlotContent<ToastContext>;
+	@ContentChild(ToastStructureDirective, {static: false}) slotStructureFromContent: ToastStructureDirective | undefined;
+
+	@Input('auSlotHeader') slotHeader: SlotContent<ToastContext>;
+	@ContentChild(ToastHeaderDirective, {static: false}) slotHeaderFromContent: ToastHeaderDirective | undefined;
+
+	/**
+	 * Callback called when the alert visibility changed.
+	 */
+	@Output('auVisibleChange') visibleChange = new EventEmitter<boolean>();
+
+	/**
+	 * Callback called when the alert is hidden.
+	 */
+	@Output('auHidden') hidden = new EventEmitter<void>();
+
+	/**
+	 * Callback called when the alert is shown.
+	 */
+	@Output('auShown') shown = new EventEmitter<void>();
+
+	readonly defaultSlots = writable(defaultConfig);
```

Comment:
Unfortunately, the type becomes invalid (at build time) if not explicitly set here.
You can reference what was done for the alert.